pkgs/by-name/ge/gemini-cli-nightly/package.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  writeShellApplication,
+  cacert,
+  curl,
+  gnused,
+  jq,
+  nix-prefetch-github,
+  prefetch-npm-deps,
+  gitUpdater,
+  ...
+}:
+buildNpmPackage (finalAttrs: {
+  pname = "gemini-cli";
+  version = "0.1.9-unstable-25-07-08";
+
+  src = fetchFromGitHub {
+    owner = "google-gemini";
+    repo = "gemini-cli";
+    rev = "821abfc456cb4664f8e7cfdc5856b1dd6564fecd";
+    hash = "sha256-2w28N6Fhm6k3wdTYtKH4uLPBIOdELd/aRFDs8UMWMmU=";
+  };
+
+  npmDeps = fetchNpmDeps {
+    inherit (finalAttrs) src;
+    hash = "sha256-yoUAOo8OwUWG0gyI5AdwfRFzSZvSCd3HYzzpJRvdbiM=";
+  };
+
+  postPatch = ''
+    for file in {,packages/cli,packages/core}package.json; do
+      substituteInPlace $file --replace-fail '"version": "0.1.5"' "version": "${finalAttrs.version}"'
```

```suggestion
      substituteInPlace $file --replace-fail '"version": "0.1.5"' '"version": "${finalAttrs.version}"'
```
I think I missed an apostrophe there, apologies :sweat_smile: